src/main/java/org/elmlang/intellijplugin/Elm.flex

```diff
@@ -180,6 +201,11 @@ RESERVED=("hiding" | "export" | "foreign" | "deriving")
         yybegin(IN_COMMENT);
         return START_COMMENT;
     }
+    {CRLF}*"{-|" {
+        commentLevel = 1;
```

Comment:
Please introduce a private method to the `%{ ... %}` section of the lexer in order not to duplicate the logic:
```java
private void startComment() {
    commentLevel = 1;
    yybegin(IN_COMMENT);
}
```